.changeset/nasty-vans-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@remix-run/react": minor
```

Comment:
We can do patch here since it's part of an unstable feature and it'll be a minor bump once we stabilize the feature

```suggestion
"@remix-run/react": patch
```